.github/workflows/deploy-git-tag.yml

```diff
@@ -0,0 +1,24 @@
+name: Deploy git tag  
+on:
+  push:
+    branches: [main, beta, alpha]
+
+jobs:
+  deploy-sdk:
+    name: Deploy git tag
+    runs-on: ubuntu-latest
+    steps:
+      - name: semantic-release setup
+        uses: levibostian/action-semantic-release-justwork@v1
```

Comment:
I think it's largely fine that we're depending on semantic-release plugins you've authored. I just want to caution that we'll want to make sure to support this work so that you (@levibostian) aren't the only engineer that can contribute fixes. There's still of course the standard OSS workflow of "fork it, open a PR from your fork, get it merged" so there's not a huge risk here. But if it's something Customer.io's teams are the primary consumers/maintainers of we might benefit from reducing the added friction of forks by having additional maintainers to the source repository (wherever they live)